hack/externalnode/install-vm.ps1

```diff
@@ -0,0 +1,196 @@
+<#
+  .SYNOPSIS
+  Installs Antrea-Agent service.
+
+  .PARAMETER NameSpace
+  ExternalNode Namespace to be used.
+
+  .PARAMETER BinaryPath
+  Specifies the path of the antrea-agent binary to be used.
+
+  .PARAMETER ConfigPath
+  Specifies the path of the antrea-agent configuration file to be used.
+
+  .PARAMETER KubeConfigDir
+  Specifies the directory which contains the antrea-agent related kubeconfig files.openvswitch
+
+  .PARAMETER InstallDir
+  The target installation directory. The default path is "C:\antrea-agent".
+#>
+Param(
+    [parameter(Mandatory = $true)] [string] $NameSpace,
+    [parameter(Mandatory = $true)] [string] $BinaryPath,
+    [parameter(Mandatory = $true)] [string] $ConfigPath,
+    [parameter(Mandatory = $true)] [string] $KubeConfigDir,
+    [parameter(Mandatory = $false)] [string] $InstallDir = "C:\antrea-agent"
+)
+
+$ErrorActionPreference = "Stop"
+
+$WorkDir = [System.IO.Path]::GetDirectoryName($myInvocation.MyCommand.Definition)
+$InstallLog = "$WorkDir\install_vm.log"
+
+# Antrea paths
+$AntreaAgentPath = [io.path]::combine($InstallDir, "antrea-agent.exe")
+$AntreaAgentConfDir = [io.path]::combine($InstallDir, "conf")
+$AntreaAgentLogDir = [io.path]::combine($InstallDir, "logs")
+$AntreaAgentConfPath = [io.path]::combine($AntreaAgentConfDir, "antrea-agent.conf")
+$AntreaAgentLogFile = [io.path]::combine($AntreaAgentLogDir, "antrea-agent.log")
+
+# Constants
+$K8Kubeconfig = "antrea-agent.kubeconfig"
+$AntreaKubeconfig = "antrea-agent.antrea.kubeconfig"
+$OVSServices = "ovsdb-server", "ovs-vswitchd"
+$AntreaAgent = "antrea-agent"
+$Kubeconfig = "kubeconfig"
+$ExternalNodeNamespace = "externalNodeNamespace"
+
+function Log($Info) {
+    $time = $(get-date -Format g)
+    "$time $Info " | Tee-Object $InstallLog -Append | Write-Host
+}
+
+function ServiceExists($ServiceName) {
+    If (Get-Service $ServiceName -ErrorAction SilentlyContinue) {
+        return $true
+    }
+    return $false
+}
+
+function CheckSupportedVersions() {
+    $script:OS_VERSION = [System.Environment]::OSVersion.Version
+    if (-Not ($OS_VERSION.Major -eq 10 -and $OS_VERSION.Minor -eq 0 -and $OS_VERSION.Build -ge 17763)) {
```

Comment:
I believe, 17763 build is the first version of Windows Server 2019. 
The check is needed because, I have not tried this on Windows Server 2016 (which have Windows 10 build, version is older that 17763)